Sources/AsyncServer.swift

```diff
@@ -1,9 +1,9 @@
 public protocol AsyncServer {
-    func serve(responder: AsyncResponder, on host: String, at port: Int) throws
+    func serve(_ responder: AsyncResponder, on host: String, at port: Int) throws
```

Comment:
I think `serve` is fine the way it is. It's clear that what is being served is what comes after the `serve`. We usually serve an app not a router. Actually the main router should be called `app` cause that's what it represents. Calling it `router` would be stating its type not its role. And adding the parameter `responder` is the same thing. Using `with` doesn't make sense also. Cause you're not serving with app. You're serving the app. Examples are better 😄. 

```swift
let app = Router { route in
    route.get("/") { request in
        return Response(body: "hello")
   }
}

try Server().serve(app, on: 8080)
```

I think this is pretty clear. 😊